Assignment1/src/test/java/models/testDeck.java

```diff
@@ -3,37 +3,74 @@
 import org.junit.Test;
 import static org.junit.Assert.assertEquals;
 
-//Created by Tanner Fry - deck test - CS361
+//Created by Tanner Fry - deck and deal test - CS361 - 01/20/2016
 
 public class testDeck{
     @Test
-    //Testing that the deck has nothing in it
+    //Testing that the deck has a full 52 cards
     public void testdeckEmpty(){
         Deck bob = new Deck();
-        assertEquals(0, bob.getCards().size());
+        assertEquals("AD", bob.getCard(51));
```

Comment:
How about an assertion that checks count==51, then if every card in the array is not null instead. This would essentially verify that there are 52 and that each one is valid. Checking if the last card in the deck is "AD" isn't enough. All other cards might not be initialized or the last card might not always be "AD".